pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -201,7 +203,7 @@ func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv0.AttestReque
 		// if the container was not located after the maximum number of attempts then the search is over.
 		if attempt >= config.MaxPollAttempts {
 			log.Warn("Container id not found; giving up")
-			return nil, k8sErr.New("no selectors found")
+			return nil, status.Error(codes.DeadlineExceeded, "no selectors found after max poll attempts")
```

Comment:
may we add attestor name as part of error?

Reply:
The facade layer adds this automatically.